resources/ory/templates/virtualservice-oathkeeper.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if .Capabilities.APIVersions.Has "networking.istio.io/v1alpha3" }}
+{{- if .Values.oathkeeper.service.api.enabled -}}
+---
```

Comment:
Why this cannot be in the templates of oathkeeper?

Reply:
as with virtualservice for oathkeeper